src/sentry/web/frontend/auth_organization_login.py

```diff
@@ -35,14 +35,14 @@ def handle_sso(self, request, organization, auth_provider):
         provider = auth_provider.get_provider()
 
         context = {
-            "CAN_REGISTER": False,
+            "CAN_REGISTER": auth_provider.flags.allow_unlinked,
             "organization": organization,
             "provider_key": provider.key,
             "provider_name": provider.name,
             "authenticated": request.user.is_authenticated(),
         }
 
-        return self.respond("sentry/organization-login.html", context)
+        return self.respond_login(request, context)
```

Comment:
This will still render the SSO login right?

Agree with dcramer that it should show some kind of text link to take them back to the form login.

Reply:
The contents of `respond_login` is exactly the same as the line I'm replacing here:
```py
return self.respond("sentry/organization-login.html", context)
```

It does render the SSO page and when `CAN_REGISTER` is true, it displays a link for registration as you both suggested.

I'll still add a test.